### PR TITLE
Group functions by namespaces.

### PR DESCRIPTION
**Pull request by j33ty**

### CODE REVIEW

File: spec/Section 10 -- Functions.md [hunk 1]
```diff
@@ -216,39 +220,7 @@ global::stringValidate(args):
 * If the length of {args} is not 1:
   * Report an error.
 
-## lower
```
Comment (atombender) [inline spec/Section 10 -- Functions.md:219]: We'll need to keep these in the global namespace.

**Review by atombender — CHANGES_REQUESTED** [review]
LGTM, except the string thing.

File: spec/Section 10 -- Functions.md (hunk 1 above)
Comment (j33ty) [reply spec/Section 10 -- Functions.md:219]: Yes, we will have to intro string namespace in a new GROQ spec version (now that we have spec versioning).

**Review by judofyr — APPROVED** [review]
LGTM!

**Pull request merged**